src/main/external-resources/UMS.conf

```diff
@@ -436,7 +436,7 @@ hide_new_media_folder =
 #   2 = Hide the file
 #   3 = Move the file to a different folder
 #   4 = Move the file to the recycle/trash bin. It will not do anything if
-#       there is no bin on the operating system
+#       this isn't possible
```

Comment:
Changed because it isn't true - it depends on the filesystem and the device, not only the OS. Windows for example have no recycler support in many cases.